notifications_utils/formatters.py

```diff
@@ -364,22 +364,34 @@ def linebreak(self):
 
     def list(self, body, ordered=True):
         return (
-            '<ol style="Margin: 0 0 20px 0; padding: 0; list-style-type: decimal;">'
+            '<table style="padding: 0 0 20px 0;">'
```

Comment:
You should add `role='presentation'` to any tables which are being used for layout (so they don’t confuse screenreaders).